user/abstractions/password.md

```diff
@@ -41,6 +41,9 @@ Set a user's **password**.
 * `password`<br>
   The **password** to set for the specified user or `False` to delete the current value.
 
+* `expired`<br>
+  A boolean toggle that is used to determine whether or not the user's **password** should be set as **expired**,  meaning that the user will be required to change their **password** on next logon. If this argument is not provided, RACF will set the user's **passpword** as **expired** by default.

```

Comment:
typo
`password` not `passpword`

Reply:
that is an embarrassing typo. Thank you for catching it.